vault/provider.go

```diff
@@ -466,6 +470,18 @@ var (
 			Resource:      ldapAuthBackendGroupResource(),
 			PathInventory: []string{"/auth/ldap/groups/{name}"},
 		},
+		"vault_nomad_secret_backend": {
+			Resource:      nomadSecretAccessBackendResource(),
+			PathInventory: []string{"/nomad"},
```

Comment:
Should `/nomad/config/access` be part of `PathInventory` here?

Reply:
@calvn Good question! Throughout the provider the mount and the config are the same resource. We derive the full path `nomad/config/access` from the `backend`. Since we derive it, `/nomad` is the default.

Reply:
`nomad/config/lease` is slightly different so I didn't add it to this resource. We _could_ but it felt different enough to have a separate resource.

Reply:
@calvn Having thought about combining `nomad/config/lease` with the other configs, I decided to keep them as two separate resources. This is simply to reduce the amount of operations create/update/read need to do for all three paths (`/nomad`, `/nomad/config/access` and `/nomad/config/lease`).

Reply:
Is `PathInventory` used for routing? Or is is just for documentation purposes?

To Calvin's question this function does seem to cover the `/nomad/config/access` endpoint?

Reply:
I can try in the morning having both but putting to the same resource handler, maybe that won't cause issues.

Reply:
Update: both can't be present because the resource name is the key and you can't duplicate it.

I changed the PathInventory to be `/nomad/config/access`. This did not have any regressions on the resources and is probably a better self-documenting change.

Reply:
PathInventory is a list, so it could also be this:

```go
		"vault_nomad_secret_backend": {
			Resource:      nomadSecretAccessBackendResource(),
			PathInventory: []string{
				"/nomad",
				"/nomad/config/access",
			},
```

But I think setting it to just "/nomad/config/access" makes sense 👍

Reply:
Oh derp, yeah, that did work! Good call!

Reply:
`/nomad`, `/nomad/config/access` and `/nomad/config/lease` are merged into a single resource `vault_nomad_secret_backend`.